### DISCUSSION TITLE
Does pyxcp support IPv6?

### DISCUSSION THREAD

**Discussion by romP44** [christoph2/pyxcp#131] [Q&A]
Hello,

I'm searching for an XCP library which support IPv6. Is it supported by pyxcp?

Romain

**Comment by christoph2** [comment]
Yes, it does.

If you are absolutely new to pyxcp use `xcphello` from [examples](https://github.com/christoph2/pyxcp/blob/master/pyxcp/examples):

```
python xcphello.py -c conf_eth.toml
```
Changes to [configuration](https://github.com/christoph2/pyxcp/blob/master/pyxcp/examples/conf_eth.toml) should be rather obvious,
`HOST` gets the correspondending `IPv6` address and IPV6 must be set to `true`.
